src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -33,3 +33,18 @@
   [number-of-accounts]
   (let [path (get-derivation-path number-of-accounts)]
     (format-derivation-path path)))
+
+(defn calculate-raw-balance
```

Comment:
feels like these are important calculations and would be good to have covered with tests so that they're behaviour does not get broken at some point.